### DISCUSSION TITLE
Add inverse to macro

### DISCUSSION THREAD

**Discussion by snipes2** [FedeClaudi/Term.jl#231] [Ideas]
I would like to see @inverse added to the styling macros, @bold @dim @italic @underline

**Comment by FedeClaudi** [comment]
what do you mean?

**Reply by snipes2** [reply]
This works:
println(
    "Make your text $(@underline("stand out"))!"
)

This doesn't:
println(
    "Make your text $(@inverse("stand out"))!"
)
